src/runtime.cc

```diff
@@ -247,7 +250,7 @@ namespace detail {
 
 	void runtime::require_call_from_application_thread() const {
 		if(std::this_thread::get_id() != m_application_thread) {
-			utils::panic("Celerity runtime, distr_queue, handler, buffer and host_object types must only be constructed, used, and destroyed from the "
+			utils::panic("Celerity runtime, queue, handler, buffer and host_object types must only be constructed, used, and destroyed from the "
```

Comment:
This should probably be "From one application thread" (the application might be multithreaded)

Reply:
We have multiple occurrences of "the application thread" in docs because Celerity is not user-thread-safe. Maybe "user thread" is an alternative? Although I'm not sure that's more helpful.